app_005/FirstViewController.swift

```diff
@@ -46,14 +46,22 @@ class FirstViewController: UIViewController, UIImagePickerControllerDelegate, UI
         if let pickedImage = info[UIImagePickerControllerOriginalImage] as? UIImage {
             imageView1.contentMode = .scaleAspectFit
             imageView1.image = pickedImage
+            let imageData = UIImagePNGRepresentation(pickedImage)
+            let imageFile = PFFile(name:"pickedImage2.png", data:imageData!)
```

Comment:
This name should use the inbuilt swift function which creates random unique alphanumeric sequesnces for names of images and such